automatic/transifex-cli/update.ps1

```diff
@@ -0,0 +1,62 @@
+﻿Import-Module AU
+Import-Module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
+
+$releases = "https://github.com/transifex/cli/releases/latest"
+
+function global:au_SearchReplace {
+  @{
+    '.\legal\VERIFICATION.txt'        = @{
+      '(?i)(^\s*location on\:?\s*)\<.*\>' = "`$1<$($Latest.ReleaseUrl)>"
+      '(?i)(\s*64\-Bit Software.*)\<.*\>' = "`$1<$($Latest.URL64)>"
+      '(?i)(\s*32\-Bit Software.*)\<.*\>' = "`$1<$($Latest.URL32)>"
+      '(?i)(^\s*checksum32\:).*'          = "`$1 $($Latest.Checksum32)"
+      '(?i)(^\s*checksum64\:).*'          = "`$1 $($Latest.Checksum64)"
+      '(?i)(^\s*checksum\s*type\:).*'     = "`$1 $($Latest.ChecksumType64)"
+    }
+    ".\$($Latest.PackageName).nuspec" = @{
+      '(?i)(?<=^\s*\<releaseNotes\>)[^<]*(?=\<\/releaseNotes\>)' = $($Latest.ReleaseUrl)
+    }
+  }
+}
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
+  $fileMap = @{
+    64 = "$PSScriptRoot\tools\x64"
+    32 = "$PSScriptRoot\tools\x86"
+  }
+
+  foreach ($architecture in 64, 32) {
+    $folder = $fileMap[$architecture]
+    $file = $Latest."FileName${architecture}"
+
+    if (-not(Test-Path $folder)) {
+      New-Item -ItemType Directory -Path $folder
+    }
+    else {
+      Get-ChildItem -Path $folder -File -Recurse -Exclude '*.zip' | Remove-Item -Force
+    }
+
+    $zip = "$PSScriptRoot\tools\$file"
+    Expand-Archive -Path $zip -Destination $folder
+    $checksum = Get-ChildItem -Path $folder -Recurse -Filter 'tx.exe' | Get-FileHash -Algorithm SHA256
+
+    $Latest."Checksum${architecture}" = $checksum.Hash
+
+    Get-ChildItem -Path $folder -File -Recurse -Exclude "*.exe" | Remove-Item -Force
+    Remove-Item $zip -Force
+  }
```

Comment:
This feels a bit wrong to me, I kinda would like to have seen the archive themself being embedded in the package, and install have an `install` script that extracted the file from it instead.

Thoughts @pauby

Reply:
I would have preferred that too because we are shipping what the release is and it's much easier for people to see that. It was just my opinion and nothing to back it up beyond my experience.

If you are of the same opinion though, let's ship what is released and do the extraction work in the `chocolateyInstall.ps1`.

Reply:
I agree, so we should extract the files in the `chocolateyInstall.ps1` script.

While there may be some rare cases we want to extract the files before creating the package, I don't believe this package is one of the cases where this is needed.

Reply:
That's fair, I'll look to get this updated for that instead.

Reply:
I've changed this around to just download the zip files and have the install script figure out which one to extract based on architecture. Tested the install seems to work, shimgen generates a shim for the extracted tx.exe as normal.